openlibrary/i18n/messages.pot

```diff
@@ -6543,8 +6543,8 @@ msgstr ""
 msgid "Inventaire.io:"
 msgstr ""
 
-#: type/author/view.html
-msgid "Links <span class=\"gray small sansserif\">(outside Open Library)"
```

Comment:
It looks like this ID is used in other .pot files. Do they need to be fixed up as well?

Reply:
Usually yes, but since the "new" also existed in the messages.pot file before this change, it should already have valid translations. The string can be deleted from the other .pot files, but it won't cause any damage, and will be deleted next time a translator updates the .pot file for their language.